models/src/commonMain/kotlin/com/revenuecat/purchases/kmp/models/VirtualCurrencies.kt

```diff
@@ -0,0 +1,19 @@
+package com.revenuecat.purchases.kmp.models
+
+/**
+ * The VirtualCurrencies object contains all the virtual currencies associated to the user.
+ */
+public class VirtualCurrencies(
```

Comment:
I guess the constructor cannot be internal because it is needed in `mappings`, right?
```suggestion
public class VirtualCurrencies internal constructor(
```

Reply:
It looks like the constructors of all of the models are public, probably for the same reason 😅

Reply:
Yea, makes sense!